src/content/docs/en/guides/images.mdx

```diff
@@ -271,7 +271,7 @@ Alternatively, the CDN may provide its own SDKs to more easily integrate in an A
 
 ## Images in Markdown files
 
-Use standard Markdown `![alt](src)` syntax in your `.md` files. This syntax works with Astro's [Image Service API](/en/reference/image-service-reference/) to optimize your local images stored in `src/`. Remote images and images stored in the `public/` folder are not optimized.
+Use standard Markdown `![alt](src)` syntax in your `.md` files. This syntax works with Astro's [Image Service API](/en/reference/image-service-reference/) to optimize your local images stored in `src/`. Images stored in the `public/` folder are not optimized.
```

Comment:
```suggestion
Use standard Markdown `![alt](src)` syntax in your `.md` files. This syntax works with Astro's [Image Service API](/en/reference/image-service-reference/) to optimize your local images stored in `src/` and remote images. Images stored in the `public/` folder are never optimized.
```
Feels like we should still mention remote images here so all the types are covered, just that now they are also optimized!